### ISSUE TITLE
Can't launch the addons manager

### ISSUE THREAD

**Issue by waldolflo** [gw2-addon-loader/GW2-Addon-Manager#34]
### Please attach a copy of config.yaml or config.ini to the issue.

**Describe the bug**
Can't launch the addons manager, even in administrator mode.

**To Reproduce**
Just clic to launch it and nothing show up

**Additional context**
I don't have any config.yaml ou config.ini in my unziped folder

**don't forget to attach the config file to the issue**

**Comment by fmmmlee** [comment]
Check that it's not being blocked by an antivirus or Windows Defender; sometimes it flags the program and won't launch it because it's not signed by a publisher.

**Comment by waldolflo** [comment]
I success to open it and install the some addons but here is the repport bug when i launch the game :

*--> Crash <--*
Assertion: Failed to load Coherent DLL: 0x7e 
File: ..\..\..\Engine\ArenaWeb\CoherentUi\AwCoh.cpp(50)
App: Gw2-64.exe 
Pid: 34792
BaseAddr: 00007FF635410000
ProgramId: 101
Build: 104333
When: 2020-06-10T09:36:13Z 2020-06-10T11:36:13+02:00
Uptime:   0 days  0:00:03
Flags: 0

*--> System <--*
Name: 34ccb1c6
IpAddr: 192.168.1.18
Processors: 8 [GenuineIntel:6:10:9]
OSVersion: Windows 10.0 (64 bit)

*--> System Memory <--*
Physical:  7973MB/16266MB  49%
Paged:     6733MB/19167MB  35%
Virtual:  134213290MB/134217727MB  3%
Load: 50%
CommitTotal:  12434MB
CommitLimit:  19167MB
CommitPeak:   17339MB
SystemCache:   7061MB
HandleCount:  127780
ProcessCount:   260
ThreadCount:   3450

*--> Process Memory <--*
Private:          101MB
WorkingSet:       101MB
PeakWorkingSet:   213MB
PageFaults:    259955

*--> Memory Category Usage (Inclusive) <--*
[Category]              [Size(MB)]  [Count]     
Root                    78.12       6228        
Uncategorized           42.26       4628        
Programmer Data         29.41       845         
I/O                     28.85       512         
Archive                 27.52       38          
Collide Havok           6.45        755         
Collide                 6.45        755         
File Service            1.24        12          
Collections             0.30        191         
Dictionary              0.24        41          
Networking              0.11        14          
Asset Service           0.09        457         
Event                   0.07        3           
Game                    0.05        56          
Engine                  0.02        12          
BTree                   0.02        16          
Content                 0.02        1           
Threading               0.01        14          
Model-Programmer        0.01        1           
CharClient              0.00        5           
Services                0.00        8           
File Core               0.00        2           
UI                      0.00        20          
Text                    0.00        1           

*--> Memory Category Usage (Exclusive) <--*
[Category]              [Size(MB)]  [Count]     
Uncategorized           42.26       4628        
Archive                 27.52       38          
Collide Havok           6.45        755         
File Service            1.24        12          
Dictionary              0.24        41          
Networking              0.11        9           
Asset Service           0.09        457         
Event                   0.07        3           
Collections             0.04        133         
Game                    0.02        46          
BTree                   0.02        16          
Content                 0.02        1           
Threading               0.01        14          
Model-Programmer        0.01        1           
Engine                  0.01        6           
CharClient              0.00        5           
Services                0.00        8           
File Core               0.00        2           
Text                    0.00        1           
Download                0.00        3           
UI                      0.00        14          
Programmer Data         0.00        2           
String                  0.00        9           
Agent                   0.00        1           

*--> DllList <--*
00007FF635410000 - C:\Program Files (x86)\Guild Wars 2\Gw2-64.exe
00007FFABE040000 - C:\WINDOWS\SYSTEM32\ntdll.dll
00007FFABC100000 - C:\WINDOWS\System32\KERNEL32.DLL
00007FFABAFD0000 - C:\WINDOWS\System32\KERNELBASE.dll
00007FFABD480000 - C:\WINDOWS\System32\USER32.dll
00007FFABB2A0000 - C:\WINDOWS\System32\win32u.dll
00007FFABDD10000 - C:\WINDOWS\System32\GDI32.dll
00007FFABB750000 - C:\WINDOWS\System32\gdi32full.dll
00007FFABB2D0000 - C:\WINDOWS\System32\msvcp_win.dll
00007FFABB370000 - C:\WINDOWS\System32\ucrtbase.dll
00007FFABDB00000 - C:\WINDOWS\System32\ADVAPI32.dll
00007FFABC1C0000 - C:\WINDOWS\System32\msvcrt.dll
00007FFABD690000 - C:\WINDOWS\System32\sechost.dll
00007FFABDE70000 - C:\WINDOWS\System32\RPCRT4.dll
00007FFABDF90000 - C:\WINDOWS\System32\WS2_32.dll
00007FFABB640000 - C:\WINDOWS\System32\WINTRUST.dll
00007FFAB41B0000 - C:\WINDOWS\SYSTEM32\VERSION.dll
00007FFABAF10000 - C:\WINDOWS\System32\MSASN1.dll
00007FFAB79B0000 - C:\WINDOWS\SYSTEM32\WTSAPI32.dll
00007FFABB470000 - C:\WINDOWS\System32\CRYPT32.dll
00007FFAB8A80000 - C:\WINDOWS\SYSTEM32\WINMM.dll
00007FFABD810000 - C:\WINDOWS\System32\PSAPI.DLL
00007FFABDBB0000 - C:\WINDOWS\System32\ole32.dll
00007FFAB8A50000 - C:\WINDOWS\SYSTEM32\winmmbase.dll
00007FFABC750000 - C:\WINDOWS\System32\combase.dll
00007FFABB5C0000 - C:\WINDOWS\System32\cfgmgr32.dll
00007FFABC070000 - C:\WINDOWS\System32\bcryptPrimitives.dll
00007FFABD9D0000 - C:\WINDOWS\System32\OLEAUT32.dll
00007FFABCD70000 - C:\WINDOWS\System32\SHELL32.dll
00007FFABD760000 - C:\WINDOWS\System32\shcore.dll
00007FFABB8F0000 - C:\WINDOWS\System32\windows.storage.dll
00007FFABAF50000 - C:\WINDOWS\System32\profapi.dll
00007FFABAF80000 - C:\WINDOWS\System32\powrprof.dll
00007FFABAF00000 - C:\WINDOWS\System32\UMPDC.dll
00007FFABDAA0000 - C:\WINDOWS\System32\shlwapi.dll
00007FFABAF30000 - C:\WINDOWS\System32\kernel.appcore.dll
00007FFABB280000 - C:\WINDOWS\System32\cryptsp.dll
00007FFA82F80000 - C:\WINDOWS\SYSTEM32\MSACM32.dll
00007FFAAFF80000 - C:\WINDOWS\WinSxS\amd64_microsoft.windows.gdiplus_6595b64144ccf1df_1.1.18362.836_none_17afac286da192bb\gdiplus.dll
00007FFA958D0000 - C:\WINDOWS\SYSTEM32\USP10.dll
00007FFAAFEC0000 - C:\WINDOWS\WinSxS\amd64_microsoft.windows.common-controls_6595b64144ccf1df_5.82.18362.836_none_2a2600ea466b531c\COMCTL32.dll
00007FFAAFF70000 - C:\WINDOWS\SYSTEM32\MSIMG32.dll
00007FFAA3F40000 - C:\WINDOWS\SYSTEM32\Secur32.dll
00007FFABA940000 - C:\WINDOWS\SYSTEM32\CRYPTBASE.DLL
00007FFABADF0000 - C:\WINDOWS\SYSTEM32\SSPICLI.DLL
00007FFABD730000 - C:\WINDOWS\System32\IMM32.DLL
00007FFAA52E0000 - C:\WINDOWS\SYSTEM32\dbghelp.dll
00007FFABB610000 - C:\WINDOWS\System32\bcrypt.dll
00007FFAB9B50000 - C:\WINDOWS\SYSTEM32\dxcore.dll
00007FFAB9250000 - C:\WINDOWS\system32\uxtheme.dll
00007FFABD460000 - C:\WINDOWS\System32\imagehlp.dll
00007FFABA280000 - C:\WINDOWS\system32\rsaenh.dll
00007FFABDD40000 - C:\WINDOWS\System32\clbcatq.dll
00007FFAB9F40000 - C:\WINDOWS\SYSTEM32\ntmarta.dll
00007FFA95530000 - C:\WINDOWS\system32\napinsp.dll
00007FFA94500000 - C:\WINDOWS\system32\pnrpnsp.dll
00007FFABA710000 - C:\WINDOWS\System32\mswsock.dll
00007FFABA460000 - C:\WINDOWS\SYSTEM32\DNSAPI.dll
00007FFABC0F0000 - C:\WINDOWS\System32\NSI.dll
00007FFABA420000 - C:\WINDOWS\SYSTEM32\IPHLPAPI.DLL
00007FFA9CB30000 - C:\WINDOWS\System32\winrnr.dll
00007FFAB7730000 - C:\WINDOWS\system32\NLAapi.dll
00007FFA9D240000 - C:\WINDOWS\system32\wshbth.dll
00007FFAB2430000 - C:\WINDOWS\System32\fwpuclnt.dll
00007FFAB1E00000 - C:\Windows\System32\rasadhlp.dll
00007FFABD820000 - C:\WINDOWS\System32\msctf.dll
0000000180000000 - C:\Program Files\Alienware\Command Center\Hook64.dll
000000006D5F0000 - C:\WINDOWS\SYSTEM32\MSVCR100.dll
00007FFAA9E50000 - C:\WINDOWS\System32\TextInputFramework.dll
00007FFAB6DF0000 - C:\WINDOWS\System32\CoreUIComponents.dll
00007FFAB8E20000 - C:\WINDOWS\System32\CoreMessaging.dll
00007FFAB6C50000 - C:\WINDOWS\SYSTEM32\wintypes.dll
00007FFAB5EC0000 - C:\WINDOWS\SYSTEM32\WindowsCodecs.dll
00007FFA9D900000 - C:\WINDOWS\system32\explorerframe.dll

*--> Game Context <--*
MapId: 0
Flags: 0x241
ElapsedTime: 00:00:01

*--> World State <--*
<WorldState />

[DbgHelp.dll is C:\WINDOWS\SYSTEM32\dbghelp.dll]
[DbgHelp.dll version 10.0.18362.1 (64/32-bit compatible)]

*--> ClientContextThreadProc Thread 0x81a0 <--*

*--> Trace <--*
Pc                Rt                : Args
00007ff6`355a2b1c 00000000`00000000 : 00000000`00000000 00000000`00000000 00000000`00000000 00000000`00000000 
00007ff6`355a2b9b 00000000`00000000 : 00000000`00000000 00000000`00000000 00000000`00000000 00000000`00000000 
00007ff6`35f39072 00000000`00000000 : 00000000`00000000 00000000`00000000 00000000`00000000 00000000`00000000 
00007ff6`358af96d 00000000`00000000 : 00000000`00000000 00000000`00000000 00000000`00000000 00000000`00000000 
00007ff6`358adec7 00000000`00000000 : 00000000`00000000 00000000`00000000 00000000`00000000 00000000`00000000 
00007ff6`35808b1b 00000000`00000000 : 00000000`00000000 00000000`00000000 00000000`00000000 00000000`00000000 
00007ff6`35807622 00000000`00000000 : 00000000`00000000 00000000`00000000 00000000`00000000 00000000`00000000 
00007ff6`35580007 00000000`00000000 : 00000000`00000000 00000000`00000000 00000000`00000000 00000000`00000000 
00007ff6`3577203e 00000000`00000000 : 00000000`00000000 00000000`00000000 00000000`00000000 00000000`00000000 
00007ff6`3577027e 00000000`00000000 : 00000000`00000000 00000000`00000000 00000000`00000000 00000000`00000000 
00007ff6`3576fef9 00000000`00000000 : 00000000`00000000 00000000`00000000 00000000`00000000 00000000`00000000 
00007ff6`3577158f 00000000`00000000 : 00000000`00000000 00000000`00000000 00000000`00000000 00000000`00000000 
00007ff6`3576f8fc 00000000`00000000 : 00000000`00000000 00000000`00000000 00000000`00000000 00000000`00000000 
00007ff6`355997e1 00000000`00000000 : 00000000`00000000 00000000`00000000 00000000`00000000 00000000`00000000 
00007ff6`35728ee9 00000000`00000000 : 00000000`00000000 00000000`00000000 00000000`00000000 00000000`00000000 
00007ffa`bc117bd4 00000000`00000000 : 00000000`00000000 00000000`00000000 00000000`00000000 00000000`00000000 
00007ffa`be0ace51 00000000`00000000 : 00000000`00000000 00000000`00000000 00000000`00000000 00000000`00000000 

*--> Thread registers <--*
rax=0000000000000001 rbx=00007ff636c3dd18 rcx=00000098372fe600
rdx=00007ff636c3dd18 rsi=0000000000000032 rdi=00000098372feb30
rip=00007ff6355a2abd rsp=00000098372fe5a0 rbp=00000098372feeb0
 r8=0000000000000032  r9=00007ff635410000 r10=00000098372fe826
r11=00000098372fea60 r12=0000000000000002 r13=0000000000000000
r14=0000000000000000 r15=0000000000000000
cs=0033 ss=002b ds=002b es=002b fs=0053 gs=002b efl=00000202

rbx-32 00007FF636C3DCF8  62655761`6e657241 6e657265`686f435c 74477741`5c744774 00000000`7070632e 
rbx +0 00007FF636C3DD18  2e2e5c2e`2e5c2e2e 5c656e69`676e455c 62655761`6e657241 6e657265`686f435c 
rbx+32 00007FF636C3DD38  6f437741`5c695574 00000070`70632e68 73676e69`74746573 6d616e65`6c69662e 
rcx-32 00000098372FE5E0  00007ff6`355a2abd 00000098`372fe5a0 00000098`372feeb0 00000007`00000021 
rcx +0 00000098372FE600  00000000`00000020 00000000`ffffffff 00000000`00000000 00000098`372fe826 
rcx+32 00000098372FE620  00000000`00000002 00000000`02000002 00009f80`0010000f 0053002b`002b0033 
rdx-32 00007FF636C3DCF8  62655761`6e657241 6e657265`686f435c 74477741`5c744774 00000000`7070632e 
rdx +0 00007FF636C3DD18  2e2e5c2e`2e5c2e2e 5c656e69`676e455c 62655761`6e657241 6e657265`686f435c 
rdx+32 00007FF636C3DD38  6f437741`5c695574 00000070`70632e68 73676e69`74746573 6d616e65`6c69662e 
 r9 +0 00007FF635410000  00000003`00905a4d 0000ffff`00000004 00000000`000000b8 00000000`00000040 
 r9+32 00007FF635410020  00000000`00000000 00000000`00000000 00000000`00000000 00000178`00000000 

*--> Code <--*
00007ff6`355a2a9d  488bf941 8bf0488d 0d36cdcc 01488bda H..A..H..6...H..
00007ff6`355a2aad  e8ae73fe ff488d4c 2460ff15 7b283e01 ..s..H.L$`..{(>.
00007ff6`355a2abd  488b8424 58010000 4c8d0534 573f0148 H..$X...L..4W?.H
00007ff6`355a2acd  89442440 4885db48 8b8424f8 00000044 .D$@H..H..$....D
00007ff6`355a2add  8bce4889 4424484c 0f45c348 8b842400 ..H.D$HL.E.H..$.
00007ff6`355a2aed  01000048 8bd74889 44245033 c9488d44 ...H..H.D$P3.H.D

*--> Stack <--*
00000098`372fe5a0  00000098`372feb51 00000000`00000200 00000000`00000021 00000000`00000000 Q./7............!...............
00000098`372fe5c0  00000098`372fe5e0 00000098`372fe600 00000000`00000000 00000098`372fe588 ../7....../7............../7....
00000098`372fe5e0  00007ff6`355a2abd 00000098`372fe5a0 00000098`372feeb0 00000007`00000021 .*Z5....../7....../7....!.......
00000098`372fe600  00000000`00000020 00000000`ffffffff 00000000`00000000 00000098`372fe826  .......................&./7....
00000098`372fe620  00000000`00000002 00000000`02000002 00009f80`0010000f 0053002b`002b0033 ........................3.+.+.S.
00000098`372fe640  00000202`002b002b 0000028a`93e70150 00000000`00000000 00000000`00000001 +.+.....P.......................
00000098`372fe660  00000000`00000010 0000028a`93e70000 0000028a`93e70000 00000000`00000001 ................................
00000098`372fe680  00000098`372fe600 00007ff6`36c3dd18 00007ff6`36c3dd18 00000098`372fe5a0 ../7.......6.......6....../7....
00000098`372fe6a0  00000098`372feeb0 00000000`00000032 00000098`372feb30 00000000`00000032 ../7....2.......0./7....2.......
00000098`372fe6c0  00007ff6`35410000 00000098`372fe826 00000098`372fea60 00000000`00000002 ..A5....&./7....`./7............
00000098`372fe6e0  00000000`00000000 00000000`00000000 00000000`00000000 00007ff6`355a2abd .........................*Z5....
00000098`372fe700  00000000`0000027f 00000000`00000000 00000000`00000000 0000ffff`00009f80 ................................
00000098`372fe720  00000000`00000000 00000000`00000000 00000000`00000000 00000000`00000000 ................................
00000098`372fe740  00000000`00000000 00000000`00000000 00000000`00000000 00000000`00000000 ................................
00000098`372fe760  00000000`00000000 00000000`00000000 00000000`00000000 00000000`00000000 ................................
00000098`372fe780  00000000`00000000 00000000`00000000 00000000`00000000 00000000`00000000 ................................
00000098`372fe7a0  00000000`00000000 00000000`00000000 00000000`00000000 00000000`00000000 ................................
00000098`372fe7c0  00000000`00000000 00000000`00000000 00000000`00000000 00000000`00000000 ................................
00000098`372fe7e0  00000000`00000000 00000000`00000000 00000000`00000000 00000000`00000000 ................................
00000098`372fe800  00000000`00000000 00000000`00000000 00000000`00000000 00000000`00000000 ................................
00000098`372fe820  00000000`00000000 00000000`00000000 00000000`00000000 00000000`00000000 ................................
00000098`372fe840  00000000`00000000 00000000`00000000 00000000`00000000 00000000`00000000 ................................
00000098`372fe860  00000000`00000000 00000000`00000000 00000000`00000000 00000000`00000000 ................................
00000098`372fe880  00000000`00000000 00000000`00000000 00000000`00000000 00000000`00000000 ................................
00000098`372fe8a0  00000000`00000000 00007ffa`be0807bd 0000028a`95e5bba0 00000034`372fec30 ........................0./74...
00000098`372fe8c0  00000000`00000004 0000028a`960e1720 0000028a`960cba40 00007ffa`2908d4f5 ........ .......@..........)....
00000098`372fe8e0  0000028a`960e4c60 00000039`02000002 00000000`00000000 0000028a`96134860 `L..........9...........`H......
00000098`372fe900  00000000`00000100 0000028a`93e70150 00000000`00000000 00000000`00000001 ........P.......................
00000098`372fe920  00000000`0000007d 0000028a`93e70000 0000028a`93e70000 00007ffa`be0860b9 }........................`......
00000098`372fe940  0000028a`93e77df0 00000000`00000600 0000028a`96135e01 00007ffa`00000080 .}...............^..............
00000098`372fe960  00000000`00000100 0000028a`9f618010 00000000`00000080 00007ffa`be081d61 ..........a.............a.......
00000098`372fe980  0000028a`93e70000 0000028a`93e70000 0000028a`9f618000 00000000`00000080 ..................a.............

*--> Error Logs <--*
09:36:13Z Failed to create THIRDPARTYSOFTWAREREADME.txt(4)
Crashed at 09:36:13Z

**Comment by fmmmlee** [comment]
Sorry for the late response. Try uninstalling the addons and then starting the game again and see if you get the same error. If not, try with each addon to find out which one is causing the issue.

**Comment by Olian04** [comment]
> I success to open it and install the some addons

what changes did you make? how did you get it to launch?

**Comment by OnaGitH444** [comment]
I had problems launching, right click, and launch as system admin...

![image](https://user-images.githubusercontent.com/74411176/99074421-56a92f00-257d-11eb-9b2c-589ac7563629.png)

**Comment by Friendly0Fire** [comment]
Tracked in #136.

**Comment by aiden201196** [comment]
hi why i cant use alt+Shift+m or T it didnt launch in game, i already update the patch to bin 64. i play on steam

**Comment by Friendly0Fire** [comment]
Make a separate issue and give actual details please.
